test/xpu_api/numerics/numeric.ops/numeric.ops.lcm/xpu_lcm.pass.cpp

```diff
@@ -0,0 +1,143 @@
+                constexpr struct
+                {
+                    int x;
+                    int y;
+                    int expect;
+                } cases[] = {{0, 0, 0}, {1, 0, 0}, {0, 1, 0},   {1, 1, 1},
+                             {2, 3, 6}, {2, 4, 4}, {3, 17, 51}, {36, 18, 36}};
+                using S1 = oneapi::dpl::make_signed_t<Input1>;
+                using S2 = oneapi::dpl::make_signed_t<Input2>;
+                using U1 = oneapi::dpl::make_unsigned_t<Input1>;
+                using U2 = oneapi::dpl::make_unsigned_t<Input2>;
+                for (auto tc : cases)
+                {
+                    { // Test with two signed types
+                        using Output = oneapi::dpl::common_type_t<S1, S2>;
+                        out[0] &= test0<S1, S2, Output>(tc.x, tc.y, tc.expect);
+                        out[0] &= test0<S1, S2, Output>(-tc.x, tc.y, tc.expect);
+                        out[0] &= test0<S1, S2, Output>(tc.x, -tc.y, tc.expect);
+                        out[0] &= test0<S1, S2, Output>(-tc.x, -tc.y, tc.expect);
+                        out[0] &= test0<S2, S1, Output>(tc.x, tc.y, tc.expect);
+                        out[0] &= test0<S2, S1, Output>(-tc.x, tc.y, tc.expect);
+                        out[0] &= test0<S2, S1, Output>(tc.x, -tc.y, tc.expect);
+                        out[0] &= test0<S2, S1, Output>(-tc.x, -tc.y, tc.expect);
+                    }
+
+                    { // test with two unsigned types
+                        using Output = oneapi::dpl::common_type_t<U1, U2>;
+                        out[0] &= test0<U1, U2, Output>(tc.x, tc.y, tc.expect);
+                        out[0] &= test0<U2, U1, Output>(tc.x, tc.y, tc.expect);
+                    }
+                    { // Test with mixed signs
+                        using Output = oneapi::dpl::common_type_t<S1, U2>;
+                        out[0] &= test0<S1, U2, Output>(tc.x, tc.y, tc.expect);
+                        out[0] &= test0<U2, S1, Output>(tc.x, tc.y, tc.expect);
+                        out[0] &= test0<S1, U2, Output>(-tc.x, tc.y, tc.expect);
+                        out[0] &= test0<U2, S1, Output>(tc.x, -tc.y, tc.expect);
+                    }
+                    { // Test with mixed signs
+                        using Output = oneapi::dpl::common_type_t<S2, U1>;
+                        out[0] &= test0<S2, U1, Output>(tc.x, tc.y, tc.expect);
+                        out[0] &= test0<U1, S2, Output>(tc.x, tc.y, tc.expect);
+                        out[0] &= test0<S2, U1, Output>(-tc.x, tc.y, tc.expect);
+                        out[0] &= test0<U1, S2, Output>(tc.x, -tc.y, tc.expect);
+                    }
+                }
+                {
+                    auto res1 = oneapi::dpl::lcm(static_cast<std::int64_t>(1234), INT32_MIN);
+                    out[0] &= (res1 == 1324997410816LL);
+                }
+            });
+        });
+    }
+
+    assert(res);
+}
+
+int
+main()
+{
+#if defined(_WIN32)
```

Comment:
Hi, @haonanya 
Please add a "TODO" here, we need to remove the macro guard once L0 backend fixes the issue.